src/validate.ts

```diff
@@ -91,18 +87,16 @@ interface ExecuteRequest{
  * @param params Array of parameters based on the respective specified commands 
  * @returns Specified format for EXECUTE intent request. 
  */
-export function generateExecuteRequest(deviceIds: Device["id"], customData: Device["customData"], commands: Command["name"], params:Command["params"]): ExecuteRequest{
+export function generateExecuteRequest(devices: Device[], execution: Command[]): ExecuteRequest{
```

Comment:
These changes look good, can you make sure to update the JSDoc above with the new parameters / types?

Reply:
Should be updated, might've not went through the first time. Lmk if you still don't see the updated JSDoc in the validate.ts folder

Reply:
I see it now.